src/reactapp/src/components/Navbar.js

```diff
@@ -290,44 +291,51 @@ const MobileNavItem = ({ label, children, href }) => { // Remove type annotation
 // Disable interface, interface are available for Typescript
 interface NavItem {
     label: string;
+    roles?: Array<string>
     subLabel?: string;
     children?: Array<NavItem>;
     href?: string;
 }*/
+const showMenuByRol = () => {
+  const userRoles = getLoggedInUserRole();
+  return NAV_ITEMS.filter(item => {
+    // Check if the item's roles match any of the roles
+    const parentMatch = item?.roles?.some(role => role === '*' || userRoles?.includes(role));
+
+    // Check if any of the item's children's roles match any of the roles
+    const childrenMatch = item?.children?.some(child => child?.roles?.some(role => role === '*' || userRoles?.includes(role)));
+
+    // Return true if either the item's roles or its children's roles match
+    return parentMatch || childrenMatch;
+  });
+};
 
 const NAV_ITEMS = [ // Remove type annotation  NAV_ITEMS: Array<NavItem>
-    //{
-    //    label: 'Inspiration',
-    //    children: [
-    //        {
-    //            label: 'Explore Design Work',
-    //            subLabel: 'Trending Design to inspire you',
-    //            href: '#',
-    //        },
-    //        {
-    //            label: 'New & Noteworthy',
-    //            subLabel: 'Up-and-coming Designers',
-    //            href: '#',
-    //        },
-    //    ],
-    //},
-    //{
-    //    label: 'Ed-Fi',
-    //    children: [
-    //        {
-    //            label: 'Teacher Evaluation',
-    //            subLabel: 'Learn the purpose of this education tool',
-    //            href: '#',
-    //        },
-    //        {
-    //            label: 'Ed-Fi Mission',
-    //            subLabel: 'Learn the core value and projects at Ed-Fi',
-    //            href: '#',
-    //        },
-    //    ],
-    //},
-    //{
-    //    label: 'Teacher Preparation Program',
-    //    href: '#',
-    //},
+  {
+    label: 'Evaluation',
```

Comment:
Can we hide or not have this section. With adding a way to get back to the main page in ETA-79, I don't think this as a menu item is desired.

Reply:
I added a redirect to the home page using the image, an I am working on the complete solution in EPPETA-79, but we have a way to get back.